frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -177,10 +187,56 @@ ListDropdown.propTypes = {
 
 const NsDropdown_ = (props) => {
   const openshiftFlag = props.flags[FLAGS.OPENSHIFT];
+
   if (flagPending(openshiftFlag)) {
     return null;
   }
-  const kind = openshiftFlag ? 'Project' : 'Namespace';
+
+  const nsModel = modelFor('Namespace');
+  const useNamespaceAccessReview = () => {
+    const canCreateNamespace = useAccessReview({
+      group: nsModel.apiGroup,
+      resource: nsModel.plural,
+      verb: 'create',
+    });
+    return canCreateNamespace;
+  };
+
+  const prModel = modelFor('Project');
+  const useProjectAccessReview = () => {
+    const canCreateProject = useAccessReview({
+      group: prModel.apiGroup,
+      resource: prModel.plural,
+      verb: 'create',
+    });
+    return canCreateProject;
+  };
```

Comment:
Use the `CAN_CREATE_PROJECT` flag instead (`useFlag` hook). Projects are a special API with a different way to determine the user's permission.